### ISSUE TITLE
A11y_Azure Tools for VS Code [BM]_Create Function App in Azure_Screen Reader:NVDA is not reading the heading for the pop up when we press enter on "Create Function App in azure" menu item.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/vscode-azurefunctions#2691]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### Actual Result:
NVDA/Narrator is not reading about check box information when we navigate to list items present inside "Filter" combo box.

### NOTE: 
No issue with Narrator

### Expected Result:
NVDA/Narrator should read about check box information when we navigate to list items present inside "Filter" combo box. It should read the state checked/un checked

### GitHubTags:
#Azure Tools for VS Code; #A11ySev2; #A11yTCS; #BM-Azure Tools for VS Code-Win-Feb2021; #DesktopApp; #Screen Reader; #WCAG1.3.1; #NVDA; #Win10; #Element:Listitem;

### OS Details:
Windows 10 Version 20H2 (OS Build 19042.685)

### Environment Details:
Application Name: Azure Tools for VS Code
Version: 1.53.1
Commit: 5d424b828ada08e1eb9f95d6cb41120234ef57c7
Date: 2021-02-08T23:29:42.785Z
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19042
Azure Function Version: 1.2.1

### Repro Steps:
Open Visual studio code
Turn on NVDA/Narrator
Tab till extension button and press enter on it.
Search for "Azure Functions" and select it.
Tab till "Install" button and press enter on it.
Tab till "Azure" button from left side pane and press enter on it.
Default screen will get open.
Tab to Tree view and use down arrow to navigate to tree items.
Press enter on Context menu and navigate to "Create Function App in Azure" list item
Press enter on it and verify NVDA is reading heading of the pop up or not.

### User Impact:
Screen Reader users will not know the functionality of controls if screen reader is not giving complete information

### Attachment for Reference:
![2691_No issue with Narrator](https://user-images.githubusercontent.com/38523250/108351711-e2c99300-720b-11eb-8b87-49f0755cd00f.png)
[2691_NVDA is not reading heading for pop up.zip](https://github.com/microsoft/vscode-azurefunctions/files/6019952/2691_NVDA.is.not.reading.heading.for.pop.up.zip)

**Comment by ejizba** [comment]
This should be filed directly on VS Code as I can repro with the sample quick input extension: https://github.com/microsoft/vscode-extension-samples/tree/main/quickinput-sample
